package.json

```diff
@@ -23,14 +23,16 @@
     "storybook": "start-storybook -p 6006",
     "storybook:build": "build-storybook -c .storybook -o public/storybook",
     "test": "jest",
-    "ts:check": "tsc --noEmit"
+    "ts:check": "tsc --noEmit",
+    "import": "npx ts-node lib/js/importers/RawColorsImporter.ts && npx ts-node lib/js/importers/TokensImporter.ts"
```

Comment:
We're importing colors when running this command for the first time. What do you think about renaming this script to `synchronizeColors`?

Reply:
This line is outdated, now is `AllInOneImporter.ts`. Yes, `synchronize` is better word so I will renamed it to `SynchronizeColorsTokens.ts`